README.md

```diff
@@ -1,124 +1,102 @@
 # Pomerium Kubernetes Ingress Controller
 
-See [docs for usage details](https://www.pomerium.com/docs/k8s/ingress.html).
+See [docs for usage details](https://www.pomerium.com/docs/k8s/ingress) for end-user details.
 
-## How it works
+# Operation Modes
 
-- Deployment
-  - containers
-    - ingress-controller
-      - Ingress controller: updates `Ingress` resources
-      - Settings controller
-        - set configuration to the databroker
-        - set configuration to the `pomerium-bootstrap`
-    - pomerium all-in-one
-      - watches `config.yaml`, mounted as a `Secret` `pomerium-bootstrap`
+- _All in one_ launches Pomerium and Ingress Controller in-process. This is easiest to use, and is recommended for most users.
+- _Controller only_ only runs ingress controller that communicates to a remote Pomerium cluster. Running Pomerium in split mode is only required to satisfy some very specific deployment requirements, and successful operation requires deep understanding of inter-component interaction. Please reach out to us first if you believe you absolutely need deploy in that mode.
```

Comment:
Where and how does a user set the operation mode?

Reply:
Then should we say so here?

Reply:
We said "get in touch with us if you want to do split mode".